command/eval.sh

```diff
@@ -110,7 +110,7 @@ _Dbg_do_eval() {
     typeset -i _Dbg_rc
     if [[ -t $_Dbg_fdi  ]] ; then
         _Dbg_set_dol_q $_Dbg_debugged_exit_code
-        . $_Dbg_evalfile >&${_Dbg_fdi}
+        . $_Dbg_evalfile >>"$_Dbg_tty"
```

Comment:
Why is appending right?

Reply:
This is actually the part that is totally hacked, since i could not figure out what to do with fdi's - I just added the actual file for testing. 

So the vscode extension uses 2 named pipes (debugger in, debugger out), they are expected to be separated from debugged script's in/out/err (command line options). I will check soon, maybe the appending is not needed...